docs/guides/projects.md

```diff
@@ -1,32 +1,137 @@
 # Working on projects
 
+uv can be used to create, install, and manage a Python project from the ground
+up.
+
 ## Creating a new project
 
+You can create a new Python project using the `uv init` command:
+
+```console
+$ uv init hello-world
+$ cd hello-world
+```
+
+Alternatively, you can initialize a project in the working directory:
+
+```console
+$ cd hello-world
+$ uv init
+```
+
+This will create the following directory structure:
+
 ```
-uv init
+.
+├── .venv
+├── pyproject.toml
+├── README.md
+├── uv.lock
+└── src
+    └── hello-world
+        └── __init__.py
```

Comment:
This is assuming that `uv init` creates the venv and lockfile. I found that this made it a little easier to explain things than having a distinction between "the things that are created by `uv init`" and "the things that are created the first time you execute `uv run`".

Reply:
I think it's fine to create them both personally. Maybe we just want an opt-out flag?